2.0/.sti/bin/assemble

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+#
+# Default STI assemble script for the ruby-2.0 image.
+#
+source "../lib/ror_context"
+
+function rake_assets_precompile() {
+  [ -f .sti/markers/disable_asset_compilation ] && return
```

Comment:
Since we are planning to have a single properties file that provides runtime environment values instead of using marker files, we should do the same for compile time.  single file (same file as for runtime) and we should just load it into the environment before running assemble.